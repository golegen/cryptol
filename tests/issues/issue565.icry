:l issue565/I.cry
