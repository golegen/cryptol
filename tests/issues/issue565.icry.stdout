Loading module Cryptol
Loading module Cryptol
Loading module T

[error] Module T does not have parameters.
